user/views.py

```diff
@@ -0,0 +1,60 @@
+import json
+import re
+import bcrypt
+
+from django.http                       import JsonResponse, HttpResponse
+from django.views                      import View
+
+from user.models                       import (
+    User
+)
+
+email_regex              = re.compile(r'^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$')
+password_regex           = re.compile(r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$')
+MINIMUM_PASSWORD_LENGTH  = 8
+
+class SignUpView(View):
+
+    def post(self,request):
+        try:
+            data         = json.loads(request.body)
+
+            email        = data['email']
+            fullname     = data['fullname']
+            password     = data['password']
+            maker_info   = data.get('maker_info', None)
```

Comment:
None 은 default 라서 굳이 필요하지 않습니다~! 하지만 가독성을 위해 일부러 추가하신거라면 굳이 수정하지 않으셔도 됩니다.
```suggestion
            maker_info   = data.get('maker_info')
```

Reply:
아닙니다 수정하겠습니다 :)!